src/components/editor/document-bar/permissions/permission-modal.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react'
+import { Modal } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { CommonModal } from '../../../common/modals/common-modal'
+import { UserAvatar } from '../../../common/user-avatar/user-avatar'
+import { PermissionList } from './permission-list'
+
+export interface PermissionsModalProps {
+  show: boolean,
+  onChangeShow: (newShow: boolean) => void
+}
+
+export interface Principal {
+  id: string
+  name: string
+  photo: string
+  canEdit: boolean
+}
+
+export const PermissionModal: React.FC<PermissionsModalProps> = ({ show, onChangeShow }) => {
+  useTranslation()
+  const [userList, setUserList] = useState<Principal[]>([{
+    id: 'dermolly',
+    photo: '/avatar.png',
+    name: 'Philip',
+    canEdit: true
+  }, {
+    id: 'emcrx',
+    photo: '/avatar.png',
+    name: 'Erik',
+    canEdit: false
+  }, {
+    id: 'mrdrogdrog',
+    photo: '/avatar.png',
+    name: 'Tilman',
+    canEdit: false
+  }])
+  const [groupList, setGroupList] = useState<Principal[]>([{
+    id: 'developer',
+    photo: '',
+    name: 'Developer',
+    canEdit: true
+  }, {
+    id: 'frontend',
+    photo: '',
+    name: 'Frontend',
+    canEdit: true
+  }])
```

Comment:
Shouldn't the user data be retreived via API calls?

Reply:
It could be retrieved either by the websocket or an http api in the future, but as we need to know when this changes immediately I guess the only logical solution would be websocket. As we currently don't mock a websocket, I just used static mock data instead.

Reply:
Why should that come via websocket? The websocket is only for live data.

Reply:
And you would classify the information that you just lost access to the note or can edit now as unlicensed data?

Reply:
I'm talking about the user information. Not the permission list itself.

Reply:
Name, Avatar, etc.

Reply:
Fair Point, das lässt sich einrichten.

Reply:
done